src/test/java/one/nio/lz4/LZ4Test.java

```diff
@@ -74,4 +74,75 @@ private int testCompression2(byte[] data) {
 
         return bytesCompressed;
     }
+
+    @Test
+    public void compressBound() {
+        Assert.assertEquals(144, LZ4.compressBound(128));
+        Assert.assertEquals(273, LZ4.compressBound(256));
+        Assert.assertEquals(4128, LZ4.compressBound(4096));
+        IllegalArgumentException e = Assert.assertThrows(IllegalArgumentException.class,
+                () -> LZ4.compressBound(Integer.MAX_VALUE));
+        Assert.assertEquals("Unsupported size: " + Integer.MAX_VALUE, e.getMessage());
+    }
+
+    @Test
+    public void compress_BoundsChecks() {
```

Comment:
Please use standart java code style with camelCase, please avoid `_` usage here and below?

As possible variant: `testCompressBoundsChecks`

Reply:
Interesting point, but it differs from used test naming approach in project